R/MixedModelsCommon.R

```diff
@@ -851,12 +843,7 @@
         if (rownames(temp_Corr)[i] == "(Intercept)") {
           var_name <- gettext("Intercept")
         } else{
-          var_name <-
-            paste(.unv(unlist(strsplit(
-              rownames(temp_Corr)[i], ":"
-            ))), collapse = ":")
-          var_name <-
-            .mmVariableNames(var_name, options$fixedVariables)
+          var_name <- .mmVariableNames(rownames(temp_Corr)[i], options$fixedVariables)
```

Comment:
tempCorr and varName

Why do we now mix this style into it?

Reply:
it was apparently always there, the only thing that changed was the line break

Reply:
You should know my opinion about snake casing by now.